AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStore/src/HierarchicalVersionUtils.dfy

```diff
@@ -44,23 +44,23 @@ module {:options "/functionSyntax:4" } HierarchicalVersionUtils {
   // TODO-HV2: Create a known answer test for CreateBKCDigest. See https://github.com/aws/aws-cryptographic-material-providers-library/commit/09a84e15b5d7311b0418180ddda69dc7314b320e
   method CreateBKCDigest (
     branchKeyContext: map<string, string>,
-    cryptoClient: AtomicPrimitives.AtomicPrimitivesClient
+    Crypto: AtomicPrimitives.AtomicPrimitivesClient
```

Comment:
Why? I also think the old name had correct naming style (camelcase)

Reply:
agreed, not sure why we changed this but let's revert it

Reply:
Because this code is almost an exact copy and paste from another method,
and it is easier for me to refactor the logic if they all use the same variables.

https://github.com/aws/aws-cryptographic-material-providers-library/blob/hv-2/hv-2/AwsCryptographicMaterialProviders/dafny/AwsCryptographicMaterialProviders/src/CanonicalEncryptionContext.dfy#L66-L82

Ideally, we would refactor all of these to ensures to a common predicate or some such,
but refactoring and abstracting code is a skill set I am still coaching Darwin and Rishav on.

Reply:
IRL: @josecorella and I agreed to take a TODO to refactor the various copies of the SHA-384 post condition into common function and reference it through out the MPL -> BKS -> BKSA.

Reply:
@josecorella I tried to make a predicate I could use everywhere,
but I could not get the input right,
so I moved on.